tests/datasets/test_ssl4eo_benchmark_landsat.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import glob
+import os
+import shutil
+from itertools import product
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pytest
+import torch
+import torch.nn as nn
+from _pytest.fixtures import SubRequest
+from pytest import MonkeyPatch
+from torch.utils.data import ConcatDataset
+
+import torchgeo.datasets.utils
+from torchgeo.datasets import SSL4EOLBenchmark
+
+
+def download_url(url: str, root: str, *args: str, **kwargs: str) -> None:
+    shutil.copy(url, root)
+
+
+class TestSSL4EOLBenchmark:
+    @pytest.fixture(
+        params=product(
+            ["tm_toa", "etm_toa", "etm_sr", "oli_tirs_toa", "oli_sr"],
+            ["cdl", "nlcd"],
+            ["train", "val", "test"],
```

Comment:
How fast is this? In other datasets we've just used zip (with items in shorter lists duplicated to make them all the same size) so that we only test 5 times instead of 5 * 2 * 3 times. If it only takes a few seconds then it doesn't really matter though.

Reply:
Running this test file takes 10-11 seconds.